ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: 'Get resource status before {{ operator }}_kuttl run'
+  ansible.builtin.shell: |
+    oc get {{ item }} > {{ cifmw_artifacts_basedir }}/logs/{{ item }}_before_{{ operator }}_kuttl.log
```

Comment:
Question I'm not sure about, do we need to set the namespace explicitely here? Sometimes it gets from the `default` namespace

Reply:
New patchset allows user to specify the proper command. we donot need to worry about the same.